_data/chains/eip155-17069.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Redstone Garnet",
+  "chain": "ETH",
+  "rpc": ["https://rpc.garnetchain.com", "wss://rpc.garnetchain.com"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "Ether",
+    "symbol": "ETH",
+    "decimals": 18
+  },
+  "infoURL": "https://redstone.xyz",
+  "shortName": "garnet",
+  "chainId": 17069,
+  "networkId": 17069,
+  "icon": "redstone",
```

Comment:
can we add the garnet chain icon so that folks connecting don't get confused?
https://redstone.xyz/chain-icons/garnet.png

Reply:
Sure, updated it to: https://ipfs.io/ipfs/QmY3ny6nsAQMK3BzQjHtVaTkgHdZTzt3qGfbg3LEgQfXun?filename=Garnet.png